src/authfile-update

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+curl https://topology.opensciencegrid.org/stashcache/authfile > /run/stashcache-cache-server-auth/Authfile.tmp
```

Comment:
This seems pretty generic.  How are we going to add other VO's to this?  I figured the query would be something like:

    https://topology.opensciencegrid.org/stashcache/authfile?cache=`hostname`

Reply:
Good point!  Addressed in such a way that the sysadmin can override the parameter (in case if the UTS name doesn't match the registration).

Reply:
Does topology actually support that query?

Reply:
Yup!

Now, it doesn't currently do anything with the information - just silently ignores it and gives the same output to each cache.